frontend/test/unit/specs/components/VMediaTag/v-media-tag.spec.js

```diff
@@ -14,9 +13,8 @@ describe("VMediaTag", () => {
   })
 
   it("should render an span tag by default", () => {
-    const wrapper = render(VMediaTag, options)
-    expect(wrapper.vm.$props.tag).toEqual("span")
-    expect(wrapper.vm.$el).toBeInstanceOf(HTMLSpanElement)
+    const { container } = render(VMediaTag, options)
+    expect(container.firstChild.tagName).toEqual("SPAN")
```

Comment:
Does `SPAN` vs `span` matter here?

Reply:
It does. The previous `render` method returns Vue-dom JavaScript objects whereas the testing-library `screen` query functions return actual (simulated via jsdom) DOM elements. There are situations when the source-casing is relevant (XHTML) but in regular HTML, the element's name is in all caps: https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName

I think that's only true for regular HTML elements, though. Custom elements (ones with names that are not part of the HTML spec), remain in their original casing, _I think_. Don't quote me on it :slightly_smiling_face: